src/server/game/Entities/Player/Player.cpp

```diff
@@ -529,6 +529,7 @@ bool Player::Create(ObjectGuid::LowType guidlow, WorldPackets::Character::Charac
     SetUInt64Value(PLAYER_FIELD_COINAGE, sWorld->getIntConfig(CONFIG_START_PLAYER_MONEY));
     SetCurrency(CURRENCY_TYPE_APEXIS_CRYSTALS, sWorld->getIntConfig(CONFIG_CURRENCY_START_APEXIS_CRYSTALS));
     SetCurrency(CURRENCY_TYPE_JUSTICE_POINTS, sWorld->getIntConfig(CONFIG_CURRENCY_START_JUSTICE_POINTS));
+    SetCurrency(CURRENCY_TYPE_ARTIFACT_KNOWLEDGE, sWorld->getIntConfig(CONFIG_CURRENCY_START_ARTIFACT_KNOWLEDGE));
```

Comment:
This is wrong (probably). 7.3 dropped currency for SMSG_ARTIFACT_KNOWLEDGE

Reply:
Can't we still use the currency server-side ?

Reply:
https://gist.github.com/DoctorKraft/0cc63fcb6b7427bbc3fa00c8735bfb74 struc are strange Oo

Reply:
structure is ArtifactCategory + KnowledgeLevel (artifact fishing pole is the second one)